src/drivers/nuttx/lv_nuttx_image_cache.c

```diff
@@ -78,6 +76,11 @@ static bool defer_init(void)
         return true;
     }
 
+    if(img_cache_p->max_size == 0) {
+        LV_LOG_INFO("Image cache is not initialized yet. Skipping deferred initialization. Because max_size is 0.");
```

Comment:
LV_LOG_WARN

Reply:
I don't think it's necessary to use warn, and if it is deliberately set to 0, it will lead to frequent printing logs.